packages/datadog-instrumentations/src/http/server.js

```diff
@@ -29,8 +31,9 @@ function wrapResponseEmit (emit) {
       return emit.apply(this, arguments)
     }
 
-    if (eventName === 'close') {
+    if (['finish', 'close'].includes(eventName) && !requestFinishedSet.has(this)) {
       finishServerCh.publish({ req: this.req })
+      requestFinishedSet.add(this)
```

Comment:
Is it `this` or `this.req`? What is `this` here?

Reply:
`this` is an instance of `ServerResponse`. I think using the request or the response here is fine, no?